### DISCUSSION TITLE
Aligning images

### DISCUSSION THREAD

**Discussion by OccultScientist** [awcodes/filament-tiptap-editor#465] [Q&A]
Hi,

Is there a way to align an image in the editor?
Ideally floating left, right or centre with text wrapping.

Thanks in advance!

**Comment by awcodes** [comment]
There's a bug with it at the moment trying find time to backport it in from the new version of the editor. Will try to get to it as soon as I can. 

Thanks.

**Reply by OccultScientist** [reply]
That would be amazing, thank you!

**Comment by ghost** [comment]
Hi, is there any updates on this?

**Reply by awcodes** [reply]
No movement on this yet. Sorry. You can position left center and right just not with text wrapping.

**Reply by ghost** [reply]
Howdy. Checking back in on this one to see if there's any movement on this.

**Reply by awcodes** [reply]
No movement. Sorry.
